File: src/main/java/com/pillcare/pillcare_server/network/RetrofitClient.java
//package com.pillcare.pillcare_server.network;
//
//import retrofit2.Retrofit;
//import retrofit2.converter.gson.GsonConverterFactory;
//
//public class RetrofitClient {
//    private static final Retrofit retrofit = new Retrofit.Builder()
//            .baseUrl("http://http://158.179.162.60:8080/api/")  // 실제 서버 IP로 교체 필요
//            .addConverterFactory(GsonConverterFactory.create())
//            .build();
//
//    public static final ApiServices apiService = retrofit.create(ApiServices.class);
//}

File: app/src/main/java/com/example/pillcare_capstone/network/RetrofitClient.kt
package com.example.pillcare_capstone.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitClient {
    private const val BASE_URL = "http://158.179.162.60:8080/"

    val apiService: ApiService by lazy {
        Retrofit.Builder()
            .baseUrl(BASE_URL)
            .addConverterFactory(GsonConverterFactory.create())
            .build()
            .create(ApiService::class.java)
    }
}